README.md

```diff
@@ -50,6 +52,11 @@ Next, the general workflow is for building and installing is:
      ``` Console
        $ CC=mpicc CXX=mpicxx FC=mpifort cmake -DNEK_DIST=none ..
      ```
+     * With NekRS on OLCF Summit:
+     ``` Console
+       $ CC=mpicc CXX=mpicxx FC=mpifort cmake -DNEK_DIST=nekrs -DCMAKE_INSTALL_LIBDIR=lib -DOCCA_CXX="g++" -DOCCA_CXXFLAGS="-O2 -ftree-vectorize -funrool-loops -mcpu=native -mtune=native" ..
```

Comment:
Here it looks like `OCCA_CXX` is g++, whereas down below it's set to xlc. Is that right?